docs/rag/adrs/template.md

```diff
@@ -0,0 +1,17 @@
+# ADR 00x: Succinct title
```

Comment:
FWIW, I really dislike serial numbers on ADRs.  Whenever there are multiple ADRs in different phases of development moving at different speeds, it creates unneeded chaos.  A team I used to be on switched from serial numbers to date stamps in the title and it made everything run much more smoothly.

Reply:
That's a good point. 

>A team I used to be on switched from serial numbers to date stamps in the title

I think that was our old team 👍 or maybe two different teams had the same idea. Anyway, the git repository will take care of tracking when something was merged and updating a date based on merge time is similarly annoying in the case that new commits dismiss approvals, so how about removing numbering entirely and letting git commit date be the sort key when needed?

Reply:
> I think that was our old team 👍 or maybe two different teams had the same idea.

Yes, I think actually the last two teams we were on together (WD and WA) both wound up adopting this.

> how about removing numbering entirely and letting git commit date be the sort key when needed

Yes, that sounds even better, thx!

Reply:
Done.